rules/library.bzl

```diff
@@ -898,7 +898,7 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
 
     objc_library(
         name = objc_libname,
-        srcs = objc_sources + objc_private_hdrs + objc_non_exported_hdrs,
+        srcs = objc_sources + objc_private_hdrs + objc_non_exported_hdrs + cpp_sources,
```

Comment:
I wonder what the the tradeoffs of this line and https://github.com/bazel-ios/rules_ios/pull/304/files#diff-23068963449a2bf4d162d813a4dda51236b9d044d1442e8a4bb4ad7680473478L867 ?

I don't know why https://github.com/bazel-ios/rules_ios/pull/304/files#diff-23068963449a2bf4d162d813a4dda51236b9d044d1442e8a4bb4ad7680473478L867 was disabled, one of the problems with putting the sources in the same library is there is no way to specify the C++ specific copts at the library level.